actions/ctf-setup-go/action.yml

```diff
@@ -40,11 +40,38 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Setup tar default options
+      shell: bash
+      # Do not overwrite existing files when extracting files from a cache archive.
+      # Since actions/cache does not support this option, we set it here as a default.
+      run: echo "TAR_OPTIONS=--skip-old-files" >> $GITHUB_ENV
+
+    - name: Get Go Version
```

Comment:
maybe just add comment saying that it's based on changes by @erikburt in `chainlink` repo and link to that original change?

Reply:
added